src/cljs/nr/utils.cljs

```diff
@@ -151,7 +151,7 @@
 (def icon-patterns
   "A sequence of icon pattern pairs consisting of an regex, used to match icon
   codes, and the span fragment that should replace it"
-  (letfn [(span-of [icon] [:span {:class (str "anr-icon " icon) :title (str " " icon)}])
+  (letfn [(span-of [icon] [:span {:class (str "anr-icon " icon) :title (str " " icon) :aria-label (str icon) :role (str "img") }])
```

Comment:
Is this right? :role (str "img")?

Reply:
I'm not sure if it's the tidiest clojure code, but it should apply the image aria role to the icons' elements. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role, I got stuck in a rabbit hole of trying to get the whole sentence to play when you hover over it (it currently stops at an icon if there is one) and that was one of the things I tried to fix it.

Reply:
Okay cool. You can just write the string "img" with the str call.

Reply:
Right, I'll go tidy that up now. I'm still geting most of my clojure knowledge from looking at the nearest piece of code haha